eta/core/video.py

```diff
@@ -195,6 +292,12 @@ def __init__(
                 each frame range when the write() method is called. When
                 out_vidpath is None or "", no videos are written
 
+            out_single_vidpath: a path like "/path/to/video.mp4" that specifies
```

Comment:
I think this should have been added in a different PR; this is rather peculiar functionality and probably deserves more discussion.  Alas, time is not too abundant to do that.

Reply:
It is awkward. But sometimes we want to process sub-clips, in which case we need `"/path/to/video/%05d-%05d.mp4"`, and sometimes we want to process the entire video, in which case we just want to say `"/path/to/video.mp4"`...

The better practice is probably to introduce `VideoClipsProcessor` to handle clips, but alas this works and is backwards compatible for now.